setup.py

```diff
@@ -42,7 +42,8 @@
     # NOTE: Maintainers, please do not require google-api-core>=2.x.x
     # Until this issue is closed
     # https://github.com/googleapis/google-cloud-python/issues/10566
-    "google-api-core >= 1.31.5, <3.0.0.dev0,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
+    "google-api-core[grpc] @ git+https://github.com/googleapis/python-api-core.git@add-api-version-header",
+    #"google-api-core >= 1.31.5, <3.0.0.dev0,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
```

Comment:
For prototyping purposes only, revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI